support.md

```diff
@@ -18,6 +23,11 @@ If you encounter a problem when you use TiDB, you can reach out for support from
     - Slack channels: [#everyone](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=docs) (English), [#tidb-japan](https://slack.tidb.io/invite?team=tidb-community&channel=tidb-japan&ref=docs) (Japanese)
     - [Stack Overflow](https://stackoverflow.com/questions/tagged/tidb) (questions tagged with #tidb)
 
++ Report a bug
+
+    - Submit an issue [on GitHub](https://github.com/pingcap/tidb/issues/new/choose)
+
 + Learn TiDB's implementation and design
 
+    - [TiDB Development Guide](https://pingcap.github.io/tidb-dev-guide/)
```

Comment:
```suggestion
    - [TiDB development guide](https://pingcap.github.io/tidb-dev-guide/)
```

Reply:
Any specific for making this lowercase? On the actual devguide it is also spelled with capitals.

Reply:
https://developers.google.com/style/capitalization#capitalization-in-references-to-titles-and-headings
> In references to any title or heading from a document that follows this guide, use sentence case even if the title or heading itself uses title case.